tests/phpunit/tests/rest-api/rest-schema-setup.php

```diff
@@ -186,6 +189,10 @@ public function test_expected_routes_in_schema() {
 			'/wp-site-health/v1/tests/authorization-header',
 			'/wp-site-health/v1/tests/page-cache',
 			'/wp-site-health/v1/directory-sizes',
+			'/wp/v2/wp_font_family',
+			'/wp/v2/wp_font_family/(?P<id>[\d]+)',
+			'/wp/v2/wp_font_family/(?P<id>[\d]+)/autosaves',
+			'/wp/v2/wp_font_family/(?P<parent>[\d]+)/autosaves/(?P<id>[\d]+)',
```

Comment:
Having fonts and wp_font_family is a little unclear which one does.

Reply:
/fonts/ are custom endpoints to install and uninstall fonts.
/wp_font_family/ are the default endpoints created by `register_custom_post_type` and are used to fetch font families data from the client.

Reply:
I followed the existing convention for global styles posts. Their custom post type is called `wp_global_styles`.

Reply:
YEah and global styles is doing it wrong too.

I see now that you updated the routes in the REST controllers, which is good. Now you need to update them here as well.